tests/Feature/Services/Forms/FormsServiceTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Tests\Feature\Services\Forms;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+use App\Eloquents\Form;
+use App\Eloquents\Question;
+use App\Eloquents\User;
+use App\Services\Forms\FormsService;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\App;
+
+class FormsServiceTest extends TestCase
+{
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->FormService = App::make(FormsService::class);
+        $this->user = factory(User::class)->create();
+        $this->form = factory(Form::class)->create([
+            'name' => 'テスト申請',
+            'is_public' => true,
+        ]);
+        $this->questions = factory(Question::class, 10)->create([
+            'form_id' => $this->form->id,
+        ]);
+    }
+
+    /**
+     * @test
+     */
+    public function 申請の複製ができるか()
```

Comment:
Service クラスのテストは `(テスト対象のメソッド名)_(テスト名)` という命名でお願いしたいです！

また、テスト名は断定形で書いているのが多いので、以下のようにするのが良いかと思います。

```suggestion
    public function copyForm_申請の複製ができる()
```

-----

↓ DownloadZipServiceTest ではこんな感じになっています。断定形になってないけど…

https://github.com/portal-dots/PortalDots/blob/87afa1366d35bb607a278c5711babdc75d9c70c6/tests/Feature/Services/Forms/DownloadZipServiceTest.php#L105